platform/view/services/db/driver/sql/sqlite/persistence.go

```diff
@@ -25,7 +25,7 @@ const sqlitePragmas = `
 
 const driverName = "sqlite"
 
-var logger = flogging.MustGetLogger("postgres-db")
+var logger = flogging.MustGetLogger("sqlite-db")
```

Comment:
Please, can you add the prefix `view-sdk.db.<the rest>`, so we can catch it more easily when setting the logs level.